modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring a namespace-scoped Argo Rollouts installation
+
+To install a namespace-scoped instance of Argo Rollouts, complete the following steps:
+
+.Prerequisites
+
+* You have logged in to the {product-title} cluster as an administrator.
+* You have installed {gitops-title} on your {product-title} cluster.
+
+.Procedure 
+
+. In the *Administrator* perspective of the web console, go to *Administration* -> *CustomResourceDefinitions*.
+. Search for `Subscription` and click the *Subscription* CRD.
+. Click the *Instances* tab and then click the *openshift-gitops-operator* subscription.
+. Click the *YAML* tab and edit the YAML file. 
++
+.. Specify the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` environment variable, with the value set to `true` in the `.spec.config.env` property.
++
+.Example of configuring the namespace-scoped Argo Rollouts installation
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+spec:
+  # (...)
+  config:
+    env:
+      - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+        value: 'true' #<1>
+----
+<1> The value set to `'true'` enables namespace-scoped installation. If the value is set to `'false'` or not specified the installation defaults to cluster-scoped mode.
+
+.. Click *Save*.
++
+The {gitops-title} Operator enables the namespace-scoped Argo Rollouts installation to reconcile the Argo Rollouts CR.
+
+. Verify that the {gitops-title} Operator has enabled the namespace-scoped Argo Rollouts installation by viewing the logs of the {gitops-shortname} container:
+.. In the *Administrator* perspective of the web console, go to *Workloads* -> *Pods*.
+.. Click the *openshift-gitops-operator-controller-manager* pod, and then click the *Logs* tab.
+.. Look for the following log statement: `Running in namespaced-scoped mode`. This statement indicates that the {gitops-title} Operator has enabled the namespace-scoped Argo Rollouts installation.
+
+. Create a `RolloutManager` resource to complete the namespace-scoped Argo Rollouts installation:
+.. Go to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps*, and click the *RolloutManager* tab.
+.. Click *Create RolloutManager*.
+.. Select *YAML view* and enter the following snippet:
++
+.Example `RolloutManager` CR for a namespace-scoped Argo Rollouts installation
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: rollout-manager
+  namespace: my-application #<1> 
+spec:
+  namespaceScoped: true
+----
+<1> Specify the name of the project where you want to install the namespace-scoped Argo Rollouts instance.
++
+.. Click *Create*. 
++
+After the `RolloutManager` CR is created, {gitops-title} begins to install the namespace-scoped Argo Rollouts instance into the selected namespace.
+
+. Verify that the namespace-scoped installation is successful.
```

Comment:
This could be the start of a verification section. 

https://redhat-documentation.github.io/modular-docs/#verification

Reply:
Agreed but as the verification section is optional I do not see any major benefit of carving out an explicit verification section. I think the content looks fine as is now. WDYT @maxwelldb?